features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1424,6 +1424,19 @@
   "console.oidc_scopes.scopes.read": ["internal_oidc_scope_mgt_view", "internal_claim_meta_view"],
   "console.oidc_scopes.scopes.update": ["internal_oidc_scope_mgt_update"],
   "console.oidc_scopes.scopes.delete": ["internal_oidc_scope_mgt_delete"],
+  "console.push_providers.enabled": true,
+  "console.push_providers.scopes.create": [
+    "internal_notification_senders_create"
+  ],
```

Comment:
@ZiyamSanthosh have we defined a feature scope for this feature?

Reply:
Currently, we have not provided the capability of feature scopes for other notification providers such as EmailProviders and SMS Providers. Hence, I have not added the feature scopes config to Push providers as well.